nbconvert/tests/files/notebook2.ipynb

```diff
@@ -1569,7 +1569,8 @@
    "metadata": {},
    "source": [
     "```python\n",
-    "def foo(bar=1):\n",
+    "import math, numpy\n",
+    "def matrix_calc(L):\n",
```

Comment:
Sorry to be nitpicky about this, but I have a suspicion that the `bar=1` keyword argument in the original function definition was intentional. Could you either make this function definition also have a keyword argument or just use the old one?